### ISSUE TITLE
k3s service start-pre times out after installation

### ISSUE THREAD

**Issue by khurtado** [rancher/k3s#360]
**Describe the bug**
When installing k3s, systemctl complains the server timed out
```
curl -sfL https://get.k3s.io | INSTALL_K3S_EXEC="--docker" sh -

systemctl status k3s
k3s.service - Lightweight Kubernetes
   Loaded: loaded (/etc/systemd/system/k3s.service; enabled; vendor preset: disabled)
   Active: failed (Result: timeout) since Tue 2019-04-16 22:40:24 UTC; 4min 10s ago
     Docs: https://k3s.io
  Process: 24638 ExecStartPre=/sbin/modprobe overlay (code=killed, signal=TERM)
  Process: 24636 ExecStartPre=/sbin/modprobe br_netfilter (code=exited, status=0/SUCCESS)

Apr 16 22:40:24 systemd[1]: Starting Lightweight Kubernetes...
Apr 16 22:40:24  systemd[1]: k3s.service start-pre operation timed out. Terminating.
Apr 16 22:40:24  systemd[1]: Failed to start Lightweight Kubernetes.
Apr 16 22:40:24  systemd[1]: Unit k3s.service entered failed state.
Apr 16 22:40:24  systemd[1]: k3s.service failed.
```

# And journalctl shows:
```
-- Defined-By: systemd
-- Support: http://lists.freedesktop.org/mailman/listinfo/systemd-devel
--
-- Unit k3s.service has begun starting up.
Apr 16 22:40:24  systemd[1]: k3s.service start-pre operation timed out. Terminating.
Apr 16 22:40:24  systemd[1]: Failed to start Lightweight Kubernetes.
-- Subject: Unit k3s.service has failed
```

**Expected behavior**
k3s service should start without issues.

**Screenshots**

Any ideas on how to debug this?
This is on CentOs 7.6, firewalld is disabled

**Comment by erikwilson** [comment]
It looks like `/sbin/modprobe overlay` is hanging for some reason, if you run that command manually as root does the same thing happen? I would think this should have been fixed with https://github.com/rancher/k3s/pull/342 tho, can you verify that TimeoutStartSec appears with `grep TimeoutStartSec /etc/systemd/system/k3s.service`?

**Comment by erikwilson** [comment]
@khurtado could you also please provide the systemd version from `systemctl --version`?

**Comment by khurtado** [comment]
@erikwilson
Overlay doesn't seem to hang 
```
$ time sudo /sbin/modprobe overlay

real	0m0.013s
user	0m0.006s
sys	0m0.007s
```

Here is my systemctl info:
```
$ systemctl --version
systemd 219
+PAM +AUDIT +SELINUX +IMA -APPARMOR +SMACK +SYSVINIT +UTMP +LIBCRYPTSETUP +GCRYPT +GNUTLS +ACL +XZ +LZ4 -SECCOMP +BLKID +ELFUTILS +KMOD +IDN
```

**Comment by erikwilson** [comment]
This should be fixed now with #365 @khurtado, if you can please retry the `curl -sfL https://get.k3s.io | ...` command and verify that k3s starts up correctly.

**Comment by khurtado** [comment]
@erikwilson That did dix that issue indeed, thanks a lot!
